example/android/app/build.gradle

```diff
@@ -60,3 +60,7 @@ android {
 flutter {
     source '../..'
 }
+
+dependencies {
+    implementation 'com.google.guava:guava:28.2-android'
```

Comment:
Is this only required in Android 24? Do you also need upgrade the version of minSDK for this example?

Reply:
There is no need to upgradation of minSDK it will work perfectly with 24.